generators/vsix-vs-win/BotBuilderVSIX-V4/BotBuilderTemplatesVSIX.csproj

```diff
@@ -1,9 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
+    <MinimumVisualStudioVersion>16.0</MinimumVisualStudioVersion>
```

Comment:
Can you help me understand this change? I understand the addition to the versions in the VSIX Manifest file, but don't understand this change. 

As I understand it, Visual Studio 2017 is "version 15". Currently it's 15.9, per this link:
https://docs.microsoft.com/en-us/visualstudio/releasenotes/vs2017-relnotes

Visual Studio 2019, when it's released, is likely to be version 16. 

Setting the Min version here to 16 means this change wouldn't work in VS 2017, correct? That doesn't seem like the right way to go.

Reply:
You're right, but lines 4 and 7-11 were done by VS2019 itself, not by me.

When opening this project file it forces a migration:
![image](https://user-images.githubusercontent.com/3463496/50968730-41c06800-14dc-11e9-875c-877764e456ab.png)

So if you want this project (not the VSIX, but the VSIX project) to be editable in VS2017 then it can't be opened in VS2019, and the other way around. For publishing the VSIX, the VSIX manifest should be leading.

Reply:
I'll revert everything but the manifest.

Reply:
I reverted the csproj.